Homework/Homework 1 and 2.cs

```diff
@@ -0,0 +1,60 @@
+﻿using Microsoft.VisualBasic;
+using System;
+using System.Diagnostics.CodeAnalysis;
+
+namespace BootCamp.Chapter
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            
+
+            int myAge = 25;
+            decimal myWeight = 77m;
```

Comment:
weight should be either float or double (I think float suits better). You don't need high precision nor big number for weight, therefore we don't care if 1/100 of mg will be dropped.

Either way, decimal is a bad choice for datatype, because it is used for high precision calculations, for example finance or scientific calculations.